jenkins/pipeline.groovy

```diff
@@ -1,3 +1,8 @@
+if (
+    (params.MTR_ARGS.contains('--big-test')) || (params.MTR_ARGS.contains('--only-big-test'))
```

Comment:
Only 8.0 understands --only-big-test option, please only check for --big-test in 5.7

Reply:
removed